### ISSUE TITLE
aria/aria-owns_attribute is now supported by Webkit

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [accessibilitysupported/a11ysupport.io#281]
I just tested aria-owns with Safari/VoiceOver, and it works, so this page should be updated https://a11ysupport.io/tech/aria/aria-owns_attribute.

* macOS version 14.3.1, Safari version 17.3.1 (19617.2.4.11.12)
* iOS version 17.3.1

Both the single and multiple reference tests pass
* https://a11ysupport.io/tests/html/aria/aria-owns-single.html
* https://a11ysupport.io/tests/html/aria/aria-owns-multiple.html

I don't know when it started working, but I know that it did not work as recently as 1.5 years ago (late 2022).

This means aria-owns is supported by all the major browser/screen reader combinations now.

**Comment by mfairchild365** [comment]
Thanks for opening this issue! I've updated the results. https://github.com/accessibilitysupported/a11ysupport.io/commit/28e4ed57733770e4baddd774de6f7c7479b24d15
